sale_order_invoicing_queued/views/queue_job_views.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2019 Tecnativa - Pedro M. Baeza
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <record id="view_queue_job_sale_order_invoicing" model="ir.ui.view">
+        <field name="model">queue.job</field>
+        <field name="priority" eval="9999"/>
```

Comment:
I'd remove this if possible.

```suggestion
```

Reply:
Not possible. This view mustn't be used by default. It's on purpose.

Reply:
Oh, that's not very obvious... A comment would help then. I didn't know it works like that.

Also do you really need 9999, or is it enough to use something more than the default (16 if I remember correctly)?

Reply:
Yes, it is. It's the normal priority field use. I take that number as a very high one for being sure the view is not used as default.